spec/plaindate.html

```diff
@@ -715,21 +708,21 @@ <h1>CreateTemporalDateFromStatic ( _constructor_, _isoYear_, _isoMonth_, _isoDay
     </emu-clause>
 
     <emu-clause id="sec-temporal-totemporaldate" aoid="ToTemporalDate">
-      <h1>ToTemporalDate ( _item_ [ , _constructor_ [ , _overflow_ ] ] )</h1>
+      <h1>ToTemporalDate ( _item_ [ , _constructor_ [ , _options_ ] ] )</h1>
       <p>
         The abstract operation ToTemporalDate returns its argument _item_ if it is already a Temporal.PlainDate instance, converts _item_ to a new Temporal.PlainDate instance if possible, and throws otherwise.
       </p>
       <emu-alg>
         1. If _constructor_ is not given, set it to %Temporal.PlainDate%.
-        1. If _overflow_ is not given, set it to *"constrain"*.
-        1. Assert: _overflow_ is either *"constrain"* or *"reject"*.
+        1. If _options_ is not given, set it to ! OrdinaryObjectCreate(%Object.prototype%).
```

Comment:
```suggestion
        1. If _options_ is not present, set it to ! OrdinaryObjectCreate(%Object.prototype%).
```
altho the editorial direction is to remove optional arguments from AOs, so it’d probably be better to provide an explicit `~empty~` in the “no options” case.

Reply:
I've made this change, but I think it would be a large change to remove all optional arguments from the Temporal spec text, so I'll leave that out of scope of this PR for now. That sounds like an editorial thing that could be done before Stage 4. Would you mind opening a new issue with more info about what is currently considered the best practice?